.pipelines/templates/e2e-pipeline-template.yml

```diff
@@ -0,0 +1,79 @@
+# ./templates/e2e-pipeline-template.yml
```

Comment:
It doesn't look like this pipeline config is getting used at the moment.  Are we adding this now for use in the future?

The Docker tools are currently getting installed via the hack/e2e/utils.sh script and the aro and e2e images will get pulled automatically by the compose command.

Reply:
You are right, removed the unwanted code